samtranslator/policy_templates_data/managed_policies.json

```diff
@@ -0,0 +1,403 @@
+{
+  "SecurityAudit": "arn:aws:iam::aws:policy/SecurityAudit",
```

Comment:
This is not deployable to all regions. This needs to be partition aware. Do you know if the names are the same across all partitions? Some services or features might be missing from other partitions as well, so some of these could be invalid there as well.

Reply:
Good catch. I'm going to shelve this for now until Authorizers are done and I'll look into making this partition-aware.